web/angular-wallet/src/app/setup/account/create-active/record/record.component.html

```diff
@@ -31,9 +31,7 @@ <h2 class="title">
     </tr>
 </table>
 <hr class="divider">
-<small>
-    Note down your passphrase. The passphrase is the master key to your account. Therefore, keep your passphrase always safe and private!
-</small>
+<small>{{ 'memorize_passphrase_help' | i18n }}</small>
```

Comment:
Switched it to use the messaging from the current BRS wallet. 
![image](https://user-images.githubusercontent.com/42594751/57577067-8c6ad600-7423-11e9-8fe2-b9bd9af00806.png)

Reply:
Maybe it should be more obvious... But we can do this later, reviewing the entire UX fir account creation